src/arch/xen/boot.S

```diff
@@ -57,21 +55,261 @@ C_ENTRY(_start):
 	addl    $0xc, %esp
 1:
 
-	//call    gdt_init
-	//call    idt_init
-
+#if 0 // TODO smp is not supported on Xen
```

Comment:
Ops, sorry I meant remove these line 
`+#if 0 // TODO smp is not supported on Xen` ...